app/acknowledgement/get-hold-category-name.js

```diff
@@ -1,5 +1,7 @@
+const holdCategoryName = require('../constants/hold-categories-names')
```

Comment:
Let's use the expansion syntax here

`const{ BANK_ACCOUNT_ANOMALY, DAX_REJECTION } = require('../constants/hold-categories-names')`